test/test_webhook_rackspace.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) LinkedIn Corporation. All rights reserved. Licensed under the BSD-2 Clause license.
+# See LICENSE in the project root for license information.
+
+from falcon import HTTPBadRequest
+
+
+def test_parse_valid_body():
+    from iris.webhooks.rackspace import rackspace
+    rs_webhook = rackspace()
+
+    fake_post = {
+        "event_id": "test_check",
+        "log_entry_id": "ntOgGC8c9G",
+        "details": {
+            "target": "www.example.com",
+            "timestamp": 1527062018051,
+            "metrics": {
+                "tt_firstbyte": {
+                    "type": "I",
+                    "data": 2,
+                    "unit": "unknown"
+                },
+                "duration": {
+                    "type": "I",
+                    "data": 2,
+                    "unit": "unknown"
+                },
+                "bytes": {
+                    "type": "i",
+                    "data": 17,
+                    "unit": "unknown"
+                },
+                "tt_connect": {
+                    "type": "I",
+                    "data": 0,
+                    "unit": "unknown"
+                },
+                "code": {
+                    "type": "s",
+                    "data": "200",
+                    "unit": "unknown"
+                }
+            },
+            "state": "CRITICAL",
+            "status": "Critical Error :-(",
+            "txn_id": ".rh-S949.h-api0.ord1.prod.cm.k1k.me.r-SJg49cHk.c-100000001.ts-1000000000001.v-cdb2afa",
+            "observations": [
+                {
+                    "monitoring_zone_id": "mzTEST1",
+                    "state": "CRITICAL",
+                    "status": "Critical Error :-(",
+                    "timestamp": 1527062018051,
+                    "collectorState": "UP"
+                },
+                {
+                    "monitoring_zone_id": "mzTEST2",
+                    "state": "WARNING",
+                    "status": "Warning :-/",
+                    "timestamp": 1527062008051
+                },
+                {
+                    "monitoring_zone_id": "mzTEST3",
+                    "state": "OK",
+                    "status": "Rocking (all good)!",
+                    "timestamp": 1527061988051
+                }
+            ]
+        },
+        "entity": {
+            "id": "enTEST",
+            "label": "Test Entity",
+            "ip_addresses": {
+                "default": "8.8.8.8"
+            },
+            "metadata": null,
```

Comment:
Need to change all of the:

`null` -> `None`
`false` -> `False`
`true` -> `True`

To convert this vaid json to valid python